lib/src/components/NxTreeView/NxTreeView.scss

```diff
@@ -18,16 +18,19 @@ $twisty-proportion-difference: 2px;
 */
 .nx-tree-view {
   box-sizing: border-box;
-  margin: var(--nx-spacing-4x) 0;
+  // left gets 7px which keeps the caret from overhanging the container
+  margin: var(--nx-spacing-4x) calc(0px - var(--nx-spacing-2x)) var(--nx-spacing-4x) calc(#{$twisty-proportion-difference / 2} - var(--nx-spacing-2x));
```

Comment:
Line too long.  Not sure what you think would be the best way to split it up

Reply:
> Line too long. Not sure what you think would be the best way to split it up

This line is only too long if we take your suggestion to use the `$twisty-proportion-difference` variable which it sounds like you've talked yourself out of above. Frankly I think that `1px` is much clearer.